File: Beecrowd/C/1134Type of Fuel.c
#include<stdio.h>
int main()
{
    int n,alcool = 0,gas = 0,disel = 0, e, i;
 
    for(i = 0;i < 100;i++)
    {
        scanf("%d", &n);
        if (n >= 1 && n <= 3){
            if(n == 1)alcool++;
            else if(n == 2)gas++;
            else if(n == 3)disel++;
        }
        
        else if(n == 4)break;
    }
    
    printf("MUITO OBRIGADO\n");
    printf("Alcool: %d\n", alcool);
    printf("Gasolina: %d\n", gas);
    printf("Diesel: %d\n", disel);
    
    return 0;
}

File: Beecrowd/Python/1134 -Type of Fuel.py
conditons = True 
alcool = 0
gas = 0 
disel = 0 

while conditons :
    T = int(input())
    if T == 4:
        conditons = False;
    else:
        if T == 1:
           alcool +=1 
        if T == 2:
           gas +=1 
        if T == 3:
           disel +=1 
print("MUITO OBRIGADO")
print(f"Alcool: {alcool}")
print(f"Gasolina: {gas}")
print(f"Diesel: {disel}")